### ISSUE TITLE
HomeKit loses connection to instance after a few days

### ISSUE THREAD

**Issue by dkrizic** [oznu/docker-homebridge#252]
I have some issues running this Docker image reliably in my Kubernetes cluster with 4 Raspberry Pi nodes running Ubuntu 64. When it starts freshly I find the Homebridge instance and can add it to HomeKit including several cameras. This works for a while very good. But after a few days the cameras stop working with "No response". 

I know that most people don't use Kubernetes, but Kubernetes (more or less) runs the given Docker images on one of the nodes. Maybe somebody has a clue what might be wrong. Here is the configuration

```
➜  ~ kubectl -n homebridge describe deployment homebridge       
Name:               homebridge
Namespace:          homebridge
Labels:             run=homebridge
Selector:           run=homebridge
Replicas:           1 desired | 1 updated | 1 total | 1 available | 0 unavailable
StrategyType:       Recreate
Pod Template:
  Labels:  run=homebridge
  Init Containers:
   homebridge-prepare:
    Image:      busybox
    Port:       <none>
    Host Port:  <none>
    Command:
      sh
      -c
      sed -e "s/LOCAL_IP/$(hostname -i)/g" </original/config.json >/homebridge/config.json; cp /original/package.json /homebridge/
    Environment:  <none>
    Mounts:
      /homebridge from homebridge-config (rw)
      /original/config.json from homebridge-config-json (rw,path="config.json")
      /original/package.json from homebridge-config-json (rw,path="package.json")
  Containers:
   homebridge:
    Image:       oznu/homebridge:arm64v8
    Ports:       51826/TCP, 8080/TCP, 10000/TCP, 10001/TCP, 10002/TCP, 10003/TCP, 31100/TCP, 31101/TCP, 31102/TCP
    Host Ports:  51826/TCP, 8080/TCP, 10000/TCP, 10001/TCP, 10002/TCP, 10003/TCP, 31100/TCP, 31101/TCP, 31102/TCP
    Limits:
      cpu:  3500m
    Requests:
      cpu:  2
    Environment:
      HOMEBRIDGE_CONFIG_UI:  1
      TZ:                    Europe/Berlin
      PACKAGES:              ffmpeg
      TERMINATE_ON_ERROR:    1
    Mounts:
      /homebridge from homebridge-config (rw)
  Volumes:
   homebridge-config-json:
    Type:      ConfigMap (a volume populated by a ConfigMap)
    Name:      homebridge
    Optional:  false
   homebridge-config:
    Type:       PersistentVolumeClaim (a reference to a PersistentVolumeClaim in the same namespace)
    ClaimName:  homebridge
    ReadOnly:   false
Conditions:
  Type           Status  Reason
  ----           ------  ------
  Progressing    True    NewReplicaSetAvailable
  Available      True    MinimumReplicasAvailable
Events:          <none>
```

In summary:

- I use host networking (the ports are directly on the node interface)
- /homebridge is a persistent volume
- /homebridge/config.json that is copied into the directory on each start from a configmap
- I run the arm64 version of the image: oznu/homebridge:arm64v8

Does anybody have an idea?
